src/parser/mod.rs

```diff
@@ -1824,18 +1824,14 @@ impl<'a> Parser<'a> {
     fn parse_big_query_struct_field_def(
         &mut self,
     ) -> Result<(StructField, MatchedTrailingBracket), ParserError> {
-        let is_anonymous_field = if let Token::Word(w) = self.peek_token().token {
-            ALL_KEYWORDS
```

Comment:
didn't make sense at all. I have replaced logic to check if we have 0, 1, 2 words in field definition.

0, 1 words -> anonymous
2 words -> named